packages/mgt-components/src/graph/graph.user.ts

```diff
@@ -80,23 +94,37 @@ export async function getMe(graph: IGraph): Promise<User> {
  * @returns {(Promise<User>)}
  * @memberof Graph
  */
-export async function getUser(graph: IGraph, userPrincipleName: string): Promise<User> {
+export async function getUser(graph: IGraph, userPrincipleName: string, requestedProps?: string[]): Promise<User> {
   const scopes = 'user.readbasic.all';
   let cache: CacheStore<CacheUser>;
 
   if (getIsUsersCacheEnabled()) {
     cache = CacheService.getCache<CacheUser>(schemas.users, schemas.users.stores.users);
     // check cache
     const user = await cache.getValue(userPrincipleName);
+    let uniqueProps;
+    if (user.user) {
```

Comment:
```suggestion
    if (user && user.user) {
```

using this test case throws an exception here:

```
<mgt-person
      user-id="2804bc07-1e1f-4938-9085-ce6d756a32d2"></mgt-person>
```

Reply:
good catch:
a003c5a